src/pages/waiting-room/waiting-room.ts

```diff
@@ -35,12 +36,18 @@ export class WaitingRoomPage extends BasePageComponent {
     public navParams: NavParams,
     public platform: Platform,
     public authenticationProvider: AuthenticationProvider,
+    private deviceProvider: DeviceProvider,
   ) {
     super(platform, navCtrl, authenticationProvider);
   }
 
   ionViewDidEnter(): void {
     this.store$.dispatch(new waitingRoomActions.WaitingRoomViewDidEnter());
+    this.deviceProvider.enableSingleAppMode();
+  }
+
+  ionViewDidLeave(): void {
+    this.deviceProvider.disableSingleAppMode();
```

Comment:
some tests around enable/disableSingleAppMode? to ensure it's not removed at any point